test/Sentry.Unity.Tests/TestHttpClientHandler.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
```

Comment:
Add namespace to go with suggestion below

```suggestion
using System.Diagnostics;
using System.Net;
```